doc/source/learning/openstack-helm.rst

```diff
@@ -0,0 +1,83 @@
+Openstack Helm
+=====================
+
+Openstack-helm is a collection of packages to deploy Opentack in kubernetes.
+
+These packages are written using the helm packaging structure called Chart that describe how to
+automatize the deployment the images.
+
+These images are created using LOCI
```

Comment:
Actually, not all images are created with LOCI, only the images for openstack services. But there are more images we are using in the deployment

Reply:
Good point.

Reply:
I agree, changed in next commit